pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: file
-version: 2.3.7
+version: 2.4.0
```

Comment:
Seems like a pretty major breaking change. Maybe start `3.0.0`?

Reply:
For users past SDK 1.23, it's actually not breaking at all, but I'm happy to bump it to `3.0.0` regardless.  Done!